mobile/apps/artwork/components/meta_data/templates/index.jade

```diff
@@ -1,12 +1,13 @@
 .artwork-section-separator
 .main-side-margin
   .artwork-meta-data-module
-    if artwork.auction
+    if artwork.auction && artwork.auction.auction_state
```

Comment:
Why the extra check for `auction_state` here?

Reply:
This is due, I think, to the nature of the sale you set up? I was noticing that it was showing 'Closed' when really there was no associated state. I figured this was safest to cover all kinds of options. 

Thoughts?

Reply:
Ahh, I'm guessing that `artwork.auction` is returning something even though in this case the artwork has a `sale` that _isn't_ an auction. Ideally, we could rename to the more generic term, so we'd be checking for `artwork.sale && artwork.sale.is_auction` but if that's a large change, maybe we can still just check for the `is_auction` boolean instead of `auction_state`?

Reply:
Updated